src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -225,10 +225,13 @@ const click = (e: MouseEvent) => {
 </script>
 
 <style lang="scss" scoped>
-.app-collection :deep(td:first-child),
-.app-collection :deep(td:first-child > *) {
+.app-collection :deep(td:first-child) {
   color: inherit;
   font-weight: $kui-font-weight-semibold;
+}
+
+.app-collection :deep(td:first-child a) {
+  color: currentColor;
```

Comment:
I don't think there are any changes required in this file. You can just leave it as it is. I think this has only ended up being changed because of the little visual changes made here which we've now undone, but we've undone them differently to how they were in the first place instead of just undoing.

Another note: `color: currentColor` is a little weird seeing as we are setting the `color` to the current color, which we are setting to the current color, which we are setting to the current color etc etc. I think `inherit` is what we mean here.

Reply:
The changes are required because otherwise the color of the name cell link for services is now blue.

I’m going to leave this as-is because it works.

Reply:
Oh I see, gotcha! Could we just do this then:

```css
.app-collection :deep(td:first-child),
.app-collection :deep(td:first-child *) {
  color: inherit;
  font-weight: $kui-font-weight-semibold;
  text-decoration: none;
}
```

i.e. remove the immediate child selector `>`? Its the removal of this in your above code that solves the issue I think not the change to use `a` or the weird `color: currentColor` (I'd rather not use that if poss, this is my biggest "what the what!?")

Reply:
`currentColor` isn’t weird. I encourage you to read up on it and let’s not dwell on PR suggestions that end up being nitpicks. This wasn’t helpful.

I’ve changed this _again_ to what you suggested.